### DISCUSSION TITLE
How do I get the children nodes of a LexicalNode?

### DISCUSSION THREAD

**Discussion by 6peterlu** [facebook/lexical#3191] [Q&A] [answered]
I only see functions to get the parent:
<img width="180" alt="image" src="https://user-images.githubusercontent.com/8334251/196048678-7a875f54-6e4b-4dcb-a007-41f5763b7070.png">

**Comment by acywatson** [comment, marked as answer]
Only ElementNodes can have children. ElementNode is a subclass of LexicalNode, like TextNode or DecoratorNode. If you have a LexicalNode, you might need to refine the type with something like ```if ($isElementNode(node)) {...}``` and then access node.getChildren.
